addons/mail/static/tests/discuss/core/public_web/sub_channels.test.js

```diff
@@ -120,6 +120,25 @@ test("create sub thread from existing message (slow network)", async () => {
     });
 });
 
+test("create sub thread from deleted message", async () => {
+    const pyEnv = await startServer();
+    const channelId = pyEnv["discuss.channel"].create({ name: "general" });
+    pyEnv["mail.message"].create({
+        body: "hey there",
+        message_type: "comment",
+        model: "discuss.channel",
+        res_id: channelId,
+    });
+    await start();
+    await openDiscuss(channelId);
+    await click(".o-mail-Message [title='Expand']");
+    await click(".o-mail-Message-moreMenu [title='Delete']");
+    await click("button", { text: "Confirm" });
+    await click(".o-mail-Message-actions [title='Expand']");
+    await click("[title='Create Thread']");
```

Comment:
`.o-mail-Message-moreMenu [title='Delete']`
=>
`.dropdown-item:contains('Delete')`

`.o-mail-Message-actions [title='Expand']`
=>
`.o-mail-Message [title='Expand']`

Also should `await contains(".o-mail-Message:contains('This message has been removed')` to ensure you create the thread after the message is effectively deleted.